Characters/mainCharacters.json

```diff
@@ -0,0 +1,72 @@
+{
+  "mainCharacters": [
+    {
+      "name": "Brute",
+      "image": "../Images/Brute.png"
```

Comment:
Would it make sense to refactor this to have some "characterImageDirectory": "../Images" property at the same level as mainCharacters?

If we wanted to change the directory structure in the future, we would have to change it for every line.

Reply:
Please feel free to reject my comment if you disagree.